sdk/log/logger.go

```diff
@@ -35,6 +34,12 @@ func newLogger(p *LoggerProvider, scope instrumentation.Scope) *logger {
 }
 
 func (l *logger) Emit(ctx context.Context, r log.Record) {
+	param := log.EnabledParameters{}
+	param.SetSeverity(r.Severity())
+	if !l.Enabled(ctx, param) {
```

Comment:
Maybe it should be possible to make it possible to filter out based on more parameters than passed via Logger.Enabled? E.g. log record body, attributes, etc.

This could be enhanced later without making any breaking change.